libpromises/cf3parse_logic.h

```diff
@@ -1084,6 +1084,7 @@ static inline void ParserCheckPromiseLine()
     // The following promise types does not require any actions
     static const char *const exceptions[] = {
         "classes", "commands", "methods", "reports", "insert_lines",
+        "packages",
```

Comment:
@nickanderson this change removes the warning generated by a promiser only package promise. I figured if I was trying to make this case "nice" I better remove the warning since it "just works" now.